views/js/runner/tpl/pageTest.tpl

```diff
@@ -1,18 +1,20 @@
-{{#if uniqid}}
-<h3>{{uniqid}}</h3>
-{{/if}}
-{{#if logo}}
+{{~#if uniqid~}}
```

Comment:
I saw in the commit message about whitespace removal, but is the `~` also intended to be added?

Reply:
It will not add `~`. Actually, this is a modifier for removing leading whitespace :)
See https://handlebarsjs.com/guide/expressions.html#whitespace-control

Reply:
Got it, thanks!